### PR TITLE
Review practice

### PR DESCRIPTION
**Pull request by alexpuente0**
In this pull we will:

- Look for correct use of semantic Tags
- Check CSS and HTML best practices
- Check linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  
+    <h1 id="title"> Bombay Bicycle Club</h1>
+    <link rel="stylesheet" href="style.css">
+    <p>4 men Indie ensamble hailing from London</p>
+    <figure id="img-div">
+      <img
+        id="image"
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e3/BombayBicycleClub.jpg/1280px-BombayBicycleClub.jpg"
+        alt="Bombay Bicycle Club playing at the University of Nottingham"
+      />
```
Comment (alexpuente0) [inline index.html:15]: - [ ]  Please add images to your repository assets, avoiding linking directly from another site.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  
+  #img-caption {
+    margin: 15px 0 5px 0;
+  }
+  
+  @media (max-width: 460px) {
+    #img-caption {
+      font-size: 1.4rem;
+    }
+  }
```
Comment (alexpuente0) [inline style.css:74]: - [ ]  Please add Media Responsiveness so the text doesn't compress when page width changes.

**Review by alexpuente0 — COMMENTED** [review]
It's a very simple project but it can still be improved.

File: style.css (hunk 2 above)
Comment (alexpuente0) [reply style.css:74]: - [ ]  Please also review linters as 2/3 were not successful.

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @alexpuente0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**